src/dsp_tools/utils/xml_utils.py

```diff
@@ -66,6 +67,22 @@ def remove_qnames_and_transform_special_tags(input_tree: etree._Element) -> etre
     return input_tree
 
 
+def transform_into_qnames(root: etree._Element) -> etree._Element:
```

Comment:
the name is wrong. Rather, the tags are transformed into localnames, but not qnames.